src/processTranslation.tsx

```diff
@@ -69,12 +71,12 @@ export const processTranslation = ({
     ...replaceableArgs
   } = args;
 
-  // Replace placeholders like `{{count}}` with values from `args`
+  // Replace placeholders like `{{someText}}` with values from `args`
   return Object.entries(replaceableArgs).reduce(
     (v, [ name, value ]) => {
       const regexp = new RegExp(`{{\\s*${name}\\s*}}`, 'g');
 
-      if (onError && !regexp.test(v)) {
+      if (onError && !regexp.test(v) && name !== COUNT) {
```

Comment:
Should we have a test covering this? (if we don't already)

Reply:
we have it, I've added https://github.com/loveholidays/phrasebook/pull/19/files#r1244012442

Reply:
Blind - thanks